components/file_attachment_list/file_attachment_list.tsx

```diff
@@ -60,6 +63,26 @@ export default function FileAttachmentList(props: Props) {
     }
 
     const postFiles = [];
+
+    const filePreviews = sortFileInfos(props.filePreviews.slice().filter((f) => !sortedFileInfos.map((s) => s.clientId).includes(f.clientId)));
+    if (filePreviews.length !== 0) {
+        postFiles.push(
+            <FilePreview
+                key='KEY'
```

Comment:
can we please use a more descriptive key value?

Reply:
That's exactly what I was thinking. `FILE_PREVIEW` would be just fine ... ideally we would have a constant with different values for this, but since we don't we should at least make sure that our keys are somewhat descriptive enough